ddtrace/contrib/langchain/patch.py

```diff
@@ -675,6 +675,72 @@ async def traced_chain_acall(langchain, pin, func, instance, args, kwargs):
     return final_outputs
 
 
+@with_traced_module
+def traced_lcecl_runnable_sequence(langchain, pin, func, instance, args, kwargs):
+    integration = langchain._datadog_integration
+    span = integration.trace(pin, "%s.%s" % (instance.__module__, instance.__class__.__name__), interface_type="chain")
+    log_inputs = {}
+    log_outputs = {}
+    try:
+        inputs = get_argument_value(args, kwargs, 0, "input")
+        if integration.is_pc_sampled_span(span):
+            log_inputs = integration.tag_lcel_inputs(span, inputs)
+        final_output = func(*args, **kwargs)
+        if integration.is_pc_sampled_span(span):
+            log_outputs = integration.tag_lcel_outputs(span, final_output)
+    except Exception:
+        span.set_exc_info(*sys.exc_info())
+        integration.metric(span, "incr", "request.error", 1)
+        raise
+    finally:
+        span.finish()
+        integration.metric(span, "dist", "request.duration", span.duration_ns)
+        if integration.is_pc_sampled_log(span):
+            integration.log(
+                span,
+                "info" if span.error == 0 else "error",
+                "sampled %s.%s" % (instance.__module__, instance.__class__.__name__),
+                attrs={
+                    "inputs": log_inputs,
+                    "outputs": log_outputs,
+                },
+            )
+    return final_output
+
+
+@with_traced_module
+async def traced_lcecl_runnable_sequence_async(langchain, pin, func, instance, args, kwargs):
```

Comment:
```suggestion
async def traced_lcel_runnable_sequence_async(langchain, pin, func, instance, args, kwargs):
```
Same request for explanatory context comment